.cargo/config

```diff
@@ -18,7 +18,7 @@ rustflags = ["-Ctarget-feature=+aes,+sse2,+ssse3"]
 
 [target.x86_64-pc-windows-msvc]
 # Enables the aes-ni instructions for RustCrypto dependency.
-# Link the C runtime statically ; https://github.com/OpenEthereum/open-ethereum/issues/6643
+# Link the C runtime statically ; https://github.com/openethereum/openethereum/issues/6643
```

Comment:
I don't think we need `crt-static` anymore do we? /cc @niklasad1

Reply:
Do you mean that we should dynamically link instead of statically link?

I don't know really but I think Microsoft recommends dynamically linking however dynamically linking might introduce runtime errors that might be hard to test in the CI.

Reply:
What I mean is that the comment seems to indicate that we added this to cater for Parity-as-a-C-library. So my question is: now that we don't have that feature in the project anymore, can't we remove this flag?
I [skimmed the RFC](https://github.com/rust-lang/rfcs/blob/master/text/1721-crt-static.md) and it seems like maybe there are reasons to keep this, but tbh I don't really know.

Reply:
I don't see any mention of parity-as-a-c-library, could you provide a link?

https://github.com/openethereum/openethereum/pull/7246#issuecomment-350323506
> I don't have a strong opinion on this. Dynamic CRT allows Microsoft to patch any security vulnerabilities that are found in the runtime. Static on the other hand reduces our support burden and simplifies installation and deployment.

Reply:
Statically linking on Windows was introduced before `parity-clib` was a thing and not related to it AFAIK.

Reply:
yup, I'm wrong and you'all are right. Sorry for the noise.